src/main/resources/public/raml/rules.raml

```diff
@@ -32,6 +32,15 @@
           body:
             application/json:
               schema: ruleCheckBody
+          responses:
+            200:
+              body:
+                application/string:
+           
+      /TestRulePageEnabled:
+        get:
+          description: This call for the current status of test rule entry point, we get status as output
+          displayName: findStatus
```

Comment:
display name should be more descriptive.

Reply:
Isn't it nice with an example of how the response can look like aswell? I find the examples very helpful (as a user/customer)